components/Careers/ApplyNewJob.vue

```diff
@@ -0,0 +1,223 @@
+                                              v-model="formData.comment"
+                                              type="textarea"
+                                              name="email"
+                                              required
+                                              autocomplete="email"
+                                            />
+                                          </FormField>                                      </div>
+                                </div>
+
+                                <div class="lg:col-span-12">
+                                        <FormField label="Upload file:" help="Max 500kb">
+                                            <FormFilePicker label="Upload" />
+                                        </FormField>
+                                </div>
+                            </div>
+                            <button type="submit" @click.prevent="submitHandler" id="submit" name="send" class="btn bg-indigo-600 hover:bg-indigo-700 border-indigo-600 hover:border-indigo-700 text-white rounded-md mt-4">Send Message</button>
+                        </form>
+                        <!--end form-->
+                    </div>
+                </div>
+            </div>
+            <!--end grid-->
+        </div>
+        <!--end container-->
+        <!-- {{ formData }} -->
+    </section>
+    <!--end section-->
+    <!-- End Section-->
+    
+
+
+
+</div>
+</template>
+
+
+<script setup>
+import { ref ,reactive } from "vue";
+import { useMainStore } from "@/stores/main";
+import {
+  mdiAccount,
+  mdiMail,
+  mdiAsterisk,
+  mdiFormTextboxPassword,
+  mdiCreditCardOutline,
+} from "@mdi/js";
+import FormField from "@/components/Forms/FormField.vue";
+import FormControl from "@/components/Forms/FormControl.vue";
+import FormFilePicker from "@/components/Forms/FormFilePicker.vue";
+// import 
+
+    const formData = ref({
+      name: "",
+      email: "",
+      phone: "",
+      jobTitle: "",
+      jobType: "",
+      comment: "",
+      uploadFile: ""
+    });
+
+    const jobTypes = ref([
+        "All Jobs","Full Time","Half Time","Remote","In Office"
+    ])
+
+    const errorMsgs = ref({
+      name: "",
+      email: "",
+      phone: "",
+      jobTitle: "",
+      jobType: "",
+      comment: "",
+      uploadFile: ""
+    });
+
+    const submitHandler = () => {
+        validateRequired("name" , formData.value.name);
+        validateRequired("jobTitle" , formData.value.jobTitle);
+        validateRequired("jobType" , formData.value.jobType);
+        validateEmail(formData.value.email);
+        validatePhone(formData.value.phone);
+    };
+
+    const validateRequired = (field,val) => {
+        if(!val ){
+            errorMsgs.value[field] = "Required field"
+        return false;
+      }
+      errorMsgs.value[field] = ""
+      return true;
+    };
+
+    const validateEmail = (email) => {        
+      const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
CAN: update regex as this will take special characters

Reply:
email regs updated